languages/ruby/ruby_on_rails.md

```diff
@@ -106,20 +121,18 @@ extract value objects (and place in app/models)
 extract service objects (and place in app/services)
 extract form objects (and place in app/forms)
 extract query objects (and place in app/models)
-introduce view objects
-extract policy objects
+introduce view objects (helpers, ViewComponents and Decorators are all good directions to think in)
 extract decorators (consider using the draper gem)
+extract policy objects (there are a number of good gems for this)
 
 # Security
 
-As a starting point, please read http://guides.rubyonrails.org/security.html and become familiar with the concepts.  We must adhere to all of the points mentioned there.
-
-Another useful, quick read is http://blog.codeclimate.com/blog/2013/03/27/rails-insecure-defaults/
+As a starting point, please read https://guides.rubyonrails.org/security.html and become familiar with the concepts.  We must adhere to all of the points mentioned there.
 
 ## Brakeman
 We include brakeman in our CI runs on both PRBuilder jobs and on commit-level jobs on develop when we can't use CodeClimate.
 
-The brakeman Jenkins job should be configured to fail if any new security vulnerabilities are found.
+The brakeman CI job should be configured to fail if any new security vulnerabilities are found.
 
 Remember to keep this gem updated to catch the latest list of vulnerabilities. 
 
```

Comment:
Do we also want to explicitly call out **Routes** as a new section for this guide? 

This might also dovetail with an explicit **API** section if so desired as it seems a lot of our latest projects have had a heavy API component attached as well.

I know for a fact that we have all been snakebit every once in a while on proper routing techniques. I know it's in the Rails guide, but might be nice to explicitly call out best-practices and confirm them here. Perfectly ok to leave it out for this draft, however.

Reply:
Rails routing can be a tricky thing.  It kinda has too many options.  I'm not sure what we'd say here other than maybe:

* use resourceful routes (when possible)
* don't overly nest

But the rails guide says both of those already so I'm not sure it deserves its own section.